plugins/data-management/src/ucsc-trackhub/model.ts

```diff
@@ -59,12 +59,8 @@ export default function UCSCTrackHubConnection(pluginManager: PluginManager) {
             ) {
               continue
             }
-            const conf = session.assemblies.find(a =>
-              readConfObject(a, 'name') === genomeName ||
-              Array.isArray(readConfObject(a, 'aliases'))
-                ? readConfObject(a, 'aliases').includes(genomeName)
-                : false,
-            )
+
+            const conf = session.assemblyManager.get(genomeName)?.configuration
```

Comment:
updated from PR #3625 which actually needed more parenthesizing in it's current form, and should be easier with this method